packages/oauth/src/index.ts

```diff
@@ -100,6 +101,36 @@ export class InstallProvider {
         throw new Error('Failed fetching data from the Installation Store');
       }
 
+      const authResult: AuthorizeResult = {};
+      authResult.userToken = queryResult.user.token;
+      authResult.teamId = queryResult.team.id;
+      if (queryResult.bot !== undefined) {
+        authResult.botToken = queryResult.bot.token;
+        authResult.botId = queryResult.bot.id;
+        authResult.botUserId = queryResult.bot.userId;
+      }
+
+      return authResult;
+    } catch (error) {
+      throw new AuthorizationError(error.message);
+    }
+  }
+
+  /**
+   * Fetches data from the installationStore for Org Installations.
+   */
+  public async orgAuthorize(source: OrgInstallationQuery): Promise<AuthorizeResult> {
+    try {
+      if (this.installationStore.fetchOrgInstallation === undefined) {
+        throw new Error('Installation Store is missing the fetchOrgInstallation method');
+      }
+
+      const queryResult = await this.installationStore.fetchOrgInstallation(source, this.logger);
+
+      if (queryResult === undefined) {
+        throw new Error('Failed fetching data from the Installation Store');
+      }
```

Comment:
@aoberoi so `queryResult.team.id` will not work here since this is an org installation (team is set to null). So for `orgAuthorize`, I assume `queryResult.team.id` is undefined/null and only check for `source.teamId`. If that exists, I add it to `authResult`